code/__defines/misc.dm

```diff
@@ -108,6 +108,8 @@
 #define AREA_ALLOW_LARGE_SIZE		0x400	// If mob size is limited in the area.
 #define AREA_BLOCK_SUIT_SENSORS		0x800	// If suit sensors are blocked in the area.
 #define AREA_BLOCK_TRACKING			0x1000	// If camera tracking is blocked in the area.
+#define TEMPERATURE_SHIELDED 		0x1200 // YW Addition: Temperature change shielding
```

Comment:
This will go wrong, you want to keep in the single bit sets. For Hexadecimal that is 
```
0x1
0x2
0x4
0x8
0x10
0x20
0x40
0x80
0x100
0x200
0x400
0x800
0x1000
0x2000
0x4000
0x8000
etc.
```

Reply:
Oh thanks for the Heads up!